changelog.md

```diff
@@ -2,6 +2,30 @@
 
 ## 1.1.0~rc1
 
+### Web Applications
+
+* Reordered and updated wording of manage.py - #4850, #4858
+* Remove python 2 support in securedrop server code - #4859
+
+### Operations
+
+* Improve specification of securedrop-app-code dependencies - #4876
+* Added check for admin password in GUI updater - #4877
```

Comment:
should be in admin tails workstation section?

Reply:
good catch, fixed